src/handle.js

```diff
@@ -1,6 +1,25 @@
 const handle = (help) => {
-    // TODO: create implementation of parse function
+    const parseSection = require('./parseSection.js');
+    const parseOption = require('./parseOption');
+    const sectionName = "Options:";
+
     let options = [];
+    parseSection(sectionName, help).forEach(section => {
+        let splitTmp = (section.split(/\n[ \t]*(-\S+?)/)).slice(1);
```

Comment:
Unnecessary `()` and move regex value to a variable, ideally that has a descriptive name what it does.